.github/workflows/create-release.yml

```diff
@@ -14,7 +14,26 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: googleapis/release-please-action@v4
+        id: release
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           config-file: release-config.json
           manifest-file: release-manifest.json
+      - uses: actions/checkout@v4
+      - name: tag major and minor versions
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        if: ${{ steps.release.outputs.releases_created == 'true' }}
+        run: |
+          set -e
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+          git remote add gh-token "https://${{ secrets.GITHUB_TOKEN }}@github.com/google-github-actions/release-please-action.git"
```

Comment:
@antroy-madetech are we adding this to release please or our repo?

Reply:
Honestly I don't know what this line is achieving. I just copied it across from the t-p-m action. The pushes all go to origin, (which is the platform-tools repo). This adds release-please as a second remote which then apparently isn't used.

I'll take it out since as far as I can tell it's not needed.